app/src/main/java/sw19/moring03/paint/Fragments/ShapeChooserFragment.java

```diff
@@ -17,12 +17,21 @@
 import sw19.moring03.paint.R;
 import sw19.moring03.paint.utils.Tool;
 
+import static sw19.moring03.paint.utils.Tool.DRAW_CHRISTMAS_TREE;
+import static sw19.moring03.paint.utils.Tool.DRAW_HEART;
+import static sw19.moring03.paint.utils.Tool.DRAW_STAR;
+import static sw19.moring03.paint.utils.Tool.DRAW_TRIANGLE;
+
 public class ShapeChooserFragment extends DialogFragment {
 
     final Item[] shapeTypes = {
             new Item("Circle", R.drawable.ic_circle_icon),
             new Item("Rectangle", R.drawable.ic_rectangle_icon),
             new Item("Oval", R.drawable.ic_oval_icon),
+            new Item("Triangle", R.drawable.ic_triangle_icon),
+            new Item("Heart", R.drawable.ic_heart_icon),
+            new Item("Star", R.drawable.ic_star_icon),
```

Comment:
the icon symbol should be exactly the same like it will then be drawn on the canvas. therefore pls  change the star icon to a pentagram icon.